### ISSUE TITLE
Deprecate bridging of private IRC channels

### ISSUE THREAD

**Issue by hellais** [TheTorProject/ooni-sysadmin#170]
Timeline:

Sep 6th 2017: @darkk My vote WRT bridges: bridge #ooni and use Slack IRC entrypoint for everything else if it's good enough. Seems, stubborn old-schoolers have lost the battle :)
Sep 6th 2017: @hellais Enabled IRC gateway
Sep 7th 2017: @hellais #ooni and #ooni-entropy stay bridged with IRC. Everything else will be accessible via the slack IRC bridge and promoted inside of the #ooni and #ooni-entropy channels?
Sep 7th 2017: @hellais actually I see there is actually only 1 real IRC user in here.
Sep 25th 2017: @hellais (on IRC): ATTENTION IRC USERS: THE SLACK BRIDGE WILL BE DISCONTINUED AND YOU WILL LOOSE MESSAGE FROM xxx, unless you use the slack to IRC gateway: https://get.slack.help/hc/en-us/articles/201727913-Connect-to-Slack-over-IRC-and-XMPP
Sep 29th 2017: hellais changed the topic of xxx to: ATTENTION IRC USERS: this channel is going to be deprecated on 2017-10-07. Refer to https://get.slack.help/hc/en-us/articles/201727913-Connect-to-Slack-over-IRC-and-XMPP to the slack IRC gateway.

On October 7th 2017 (more than 1 month after the first notice). All other channels apart from:

* #ooni
* #ooni-dev
* #ooni-entropy

Will be reachable **only** via the slack IRC bridge. Adjust your clients accordingly 🎉

**Comment by hellais** [comment]
Note: we have not actually deprecated this yet, but it's going to happen soon. @anadahz I think you are currently the only one that is actually using the slack-to-irc bridge for private channels.

**Comment by anadahz** [comment]
Issue status changed to: resolved

**Comment by hellais** [comment]
> Issue status changed to: resolved

Did you re-deploy slack-irc and remove the slack bridge?

**Comment by anadahz** [comment]
> Did you re-deploy slack-irc and remove the slack bridge?

No, I don't see this mentioned in this issue, perhaps this issue should be updated to reflect the requested changes or add a new issue to `re-deploy slack-irc and remove the slack bridge`.

**Comment by hellais** [comment]
> No, I don't see this mentioned in this issue, perhaps this issue should be updated to reflect the requested changes or add a new issue to re-deploy slack-irc and remove the slack bridge

Ok, then this issue is not resolved. I did mention it: "Will be reachable only via the slack IRC bridge"

**Comment by hellais** [comment]
```
08:47 -!- hellais changed the topic to: 2017-10-24, may this channel RIP
```
